SYNC.md

```diff
@@ -0,0 +1,143 @@
+# Automerge Sync Protocol Documentation
+
+The automerge network sync protocol is designed to bring two documents into sync by exchanging messages between peers until both documents they have the same contents. It does this by exchanging rounds of sync messages. These sync messages contain two parts: 
+ * a lossily-compressed list of changes it already has (implicitly requesting the remainder)
+ * changes it believe the other peer needs
```

Comment:
I suppose we aren't wanting to mention the explicit request for changes?

Reply:
How would you reword this? I figured it would be easier to save a discussion of what goes into the messages until later on under the "how it works" section.

Reply:
(Sorry for not getting back earlier) I think it works fine without it and can go into a more detailed section later.